l2geth/cmd/utils/flags.go

```diff
@@ -1660,6 +1665,9 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	if ctx.GlobalIsSet(RPCGlobalGasCap.Name) {
 		cfg.RPCGasCap = new(big.Int).SetUint64(ctx.GlobalUint64(RPCGlobalGasCap.Name))
 	}
+	if ctx.GlobalIsSet(RPCGlobalEVMTimeoutFlag.Name) {
```

Comment:
Is this line triggered always so that the default value is set? If not then it will change functionality, the default will be 0

Reply:
Ok I see the default value is set in another file

Reply:
I tried to keep the pattern from Geth